src/index.js

```diff
@@ -1,34 +1,77 @@
 import './index.css';
 
-const tasks = [
-  {
-    description: 'to cook',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'to make coffee',
-    completed: false,
-    index: 1,
-  },
-];
-
-const loadList = (tasks) => {
+const toDoList = document.getElementById('todolist');
+const loadTask = (tasks) => {
   tasks.sort((a, b) => a.index - b.index);
-  const toDoList = document.getElementById('todolist');
+
   toDoList.innerHTML = '';
   tasks.forEach((task) => {
     const template = ` <li class= "border-bottom border-opacity-10 p-3 d-flex justify-content-between">
                           <div>
                           <input type="checkbox" class="me-2 mt-2" />
                           ${task.description}
                           </div>
-                          <div><div class="ellipsis"><button class="btn"><i class="fa-solid fa-ellipsis-vertical"></i></button></div>
-                          <div class="trash"></i><i class="fa-solid fa-trash-can"></i></div>
+                          <div>
+                          <div class="ellipsis">
+                            <button class="btn btn-ellipsis" data-index="${task.index}">
+                              <i class="fa-solid fa-ellipsis-vertical"></i>
+                            </button>
+                          </div>
+                          <div class="hidden">
+                            <button class="btn btn-trash" data-index="${task.index}">
+                              <i class="fa-solid fa-trash-can"></i>
+                            </button>
                           </div>
+                        </div>
                        </li>`;
     toDoList.innerHTML += template;
   });
 };
 
-loadList(tasks);
+const addTask = () => {
+  const add = document.getElementById('add').value;
+  const tasks = JSON.parse(localStorage.getItem('tasks') || JSON.stringify([]));
+
+  const newTask = {
+    description: add,
+    completed: false,
+    index: tasks.length + 1,
+  };
+
+  tasks.push(newTask);
+  loadTask(tasks);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const removeTask = (value) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  const filtered = tasks.filter((task) => task.index !== parseInt(value, 10));
+  filtered.forEach((task, i) => {
+    task.index = i + 1;
+  });
+  localStorage.setItem('tasks', JSON.stringify(filtered));
+  loadTask(filtered);
+};
```

Comment:
- [ ] Your project requirements state that `Create a new JavaScript file for the new functionality`. Please define the functions for add, deleting & editing in this new file 👍🏾

Reply:
- [ ] There is no functionality for editing in your project. Kindly add one 🦖